src/test/resources/features/per_project_configuration.feature

```diff
@@ -26,7 +26,16 @@ Feature: Per-project configuration
 		Then I see "Using groovy version 2.4.1 in this shell."
 		And the candidate "groovy" version "2.4.1" should be in use
 		And the candidate "groovy" version "2.0.5" should be the default
-
+		
+	Scenario: The env install subcommand is issued with an sdkman project configuration present
+		Given the system is bootstrapped
+	    And the file ".sdkmanrc" exists and contains "groovy=2.4.1"
+		And the candidate "groovy" version "2.4.1" is available for download
+		When I enter "sdk env install"
+		Then I see "Done installing!"
+		And the candidate "groovy" version "2.4.1" is installed
+		And the candidate "groovy" version "2.4.1" should not be the default
```

Comment:
Should we make mention that there is already a default version installed like in line 21?

Reply:
Good idea!